serverscripts/docker.py

```diff
@@ -21,6 +57,72 @@ def is_docker_available():
     return os.path.exists("/etc/docker")
 
 
+def python_details(container):
+    """Run pip freeze in containers that are python-based
+
+    A container is python based if it has "python" in its command.
+    """
+    # identify the python interpreter inside the docker
+    split_command = container["command"].strip('"').split(" ")
+    for python_exec in PYTHON_EXEC_OPTIONS:
+        if python_exec in split_command:
+            break
+    else:
+        # it is some other command (gunicorn; bin/gunicorn)
+        dirname = os.path.dirname(split_command[0])
+        if dirname == "":
+            python_exec = "python3"  # global default
+        else:
+            python_exec = os.path.join(dirname, "python")
+    python_in_docker = "docker exec " + container["id"] + " " + python_exec + " "
```

Comment:
Volgens mij is dit in principe veilig en generiek genoeg.
Wel is het af en toe zo dat er een docker een beetje hangt: benieuwd hoe 'ie daarmee om gaat. Maar dat merken we snel genoeg op de p-project-task-d1 :-)